docs/workflows/sync.rst

```diff
@@ -0,0 +1,106 @@
+Synchronize a Repository
+========================
+
+Users can populate their repositories with content from an external source like PyPI by syncing
+their repository.
+
+Create a Repository
+-------------------
+
+Start by creating a new repository named "foo"::
+
+    $ http POST http://localhost:8000/pulp/api/v3/repositories/ name=foo
+
+Response::
+
+    {
+        "_href": "http://localhost:8000/pulp/api/v3/repositories/e81221c3-9c7a-4681-a435-aa74020753f2/",
+        ...
+    }
+
+If you want to copy/paste your way through the guide, create an environment variable for the repository URI.::
+
+    $ export REPO_HREF=$(http :8000/pulp/api/v3/repositories/ | jq -r '.results[] | select(.name == "foo") | ._href')
+
+
+Create a Remote
+---------------
+
+Creating a remote object informs Pulp about an external content source. In this case, we will be
+using a fixture, but Python remotes can be anything that implements the PyPI API. This can be PyPI
+itself, a fixture, or even an instance of Pulp 2. You can use any Python remote to sync content
+into any repository.::
+
+
+    $ http POST $PULP_BASE/pulp/api/v3/remotes/python/ \
+        name='bar' \
+        url='https://repos.fedorapeople.org/repos/pulp/pulp/fixtures/python-pypi/' \
+        projects='["shelf-reader"]'
+
+Response::
+
+    {
+        "_href": "http://localhost:8000/pulp/api/v3/repositories/foo/remotes/python/3750748b-781f-48df-9734-df014b2a11b4/",
+        ...
+    }
+
+Again, you can create an environment variable for convenience::
+
+    $ export REMOTE_HREF=$(http $PULP_BASE/pulp/api/v3/remotes/python/ | jq -r '.results[] | select(.name == "bar") | ._href')
+
+
+Sync repository foo with remote
+-------------------------------
+
+Use the remote object to kick off a synchronize task by specifying the repository to
+sync with. You are telling pulp to fetch content from the remote and add to the repository::
+
+    $ http POST $REMOTE_HREF'sync/' repository=$REPO_HREF
+
+Response::
```

Comment:
I don't care so much about this, but it would be nice if we had all of the "Response" sections using ```localhost``` instead of some using ```localhost``` and some using ```dev.pulp3```.